.env.example

```diff
@@ -6,6 +6,7 @@ ADDRESS=<Some address such as 127.0.0.1>
 #front-end server
 FRONTEND_ADDRESS_DEV=localhost:1337
 FRONTEND_ADDRESS_DEPLOY=app.mchacks.ca
+FRONTEND_ADDRESS_BETA=develop--mchacks-dashboard.netlify.com
```

Comment:
can you remove this for now? I think it's better to discuss w everyone first and we can always add it back easily